spatial-lib/tests.edn

```diff
@@ -15,11 +15,14 @@
  :color? #profile {:default true
                    :ci false}
 
- :reporter #profile {:default kaocha.report.progress/report
-                     :ci kaocha.report/documentation}
+ :reporter kaocha.report/documentation
 
  :kaocha.plugin.junit-xml/target-file "target/junit.xml"
 
+ ;; re-enable when cloverage is functional for this module
```

Comment:
Be more specific on why cloverage is not working for spatial-lib in the comment here.

Reply:
There is already a separate ticket tracking this problem.